src/app/src/components/FacilityListItemsTableRow.jsx

```diff
@@ -52,7 +59,17 @@ const FacilityListItemsTableRow = memo(({
             padding="none"
             style={listTableCellStyles.nameCellStyles}
         >
-            {name}
+            {
+                (newFacility && oarID)
+                    ? (
+                        <Link
+                            to={makeFacilityDetailLink(oarID)}
+                            href={makeFacilityDetailLink(oarID)}
```

Comment:
Is `href` here specified as a fallback?

Reply:
`href`'s added here because Eslint requires placing it on these `Link` tags, although does feel redundant to have both props.